### ISSUE TITLE
Greendao Query Error 

### ISSUE THREAD

**Issue by rootmonty** [greenrobot/greenDAO#519]
Hey everyone,I am trying to execute a query operation to secure a single unique value from a database using queryBuilder.where(queryBuilder.and(XYZ,ABC)) (* for example) .Querybuilder is giving a list and therfore I am picking the zeroth element of it(Technicall analogous to executeSingle() operation in activeandroid)
But I am not getting the correct queries value.Can someone assist me through this. 
Basically I am migrating from active android to Greendao for fast querying.
Active android provides facility to get a single unique object in database using executeSingle().Does Greendao has anything of that sort?

**Comment by greenrobot-team** [comment]
Check out the second paragraph of the [Query and LazyList section](http://greenrobot.org/greendao/documentation/queries/#Query_and_LazyList) of the docs.

`.unique()` or `.uniqueOrThrow()` should do what you want. -ut

**Comment by rootmonty** [comment]
Can u show me an example.that would be very generous of u

**Comment by greenrobot-team** [comment]
Again, [see the documentation](http://greenrobot.org/greendao/documentation/queries/). Adapting the first example:
```java
User joe = userDao.queryBuilder()
  .where(Properties.FirstName.eq("Joe"))
  .build()
  .unique();
```
-ut
